components/km271_wifi/number.py

```diff
@@ -50,8 +56,42 @@
                 cv.Optional(CONF_MIN_VALUE, default=10): cv.float_,
                 cv.Optional(CONF_STEP, default=0.5): cv.positive_float,
             }),
-            
-
+            cv.Optional(CONF_HC1_ROOMT_TARGET_OFFSET): number.NUMBER_SCHEMA.extend({
+                cv.GenerateID(): cv.declare_id(BuderusParamNumber),
+                cv.Optional(CONF_MAX_VALUE, default=5): cv.float_,
+                cv.Optional(CONF_MIN_VALUE, default=-5): cv.float_,
+                cv.Optional(CONF_STEP, default=1): cv.float_,
```

Comment:
The parameter can be written in steps of 0.5 to the controller - so I'd suggest allowing this via home assistant, too
```suggestion
                cv.Optional(CONF_STEP, default=0.5): cv.float_,
```

Reply:
Is this the case when you change offset manually? I can not confirm this in my setup changing hc2 offset manually.

Reply:
I just can change it by 1°C

Reply:
I just tried it. The control unit only displays whole numbers and performs steps of 1 degree.
However, with a step size of 0.5 here, everything else works:
* If I set the value to 1.5 and reboot, 1.5 is shown in Home Assistant
* If I set the value to 1.5 the values for boiler activation temperature are different from the ones at 1.0 and 2.0.

So I would consider this step size a feature that's working and just not available using manual control.

Reply:
ok, let's give it a try